vendor/github.com/containerd/continuity/sysx/xattr_unsupported.go

```diff
@@ -1,4 +1,4 @@
-// +build freebsd solaris
+// +build openbsd freebsd solaris
```

Comment:
Are there pull requests for these changes upstream? Otherwise vendor-ci will keep failing :smile:

```
 D vendor/github.com/containerd/continuity/sysx/xattr_openbsd.go
 M vendor/github.com/containerd/continuity/sysx/xattr_unsupported.go
 M vendor/github.com/docker/docker/pkg/term/winsize.go
 D vendor/github.com/docker/docker/pkg/term/winsize_openbsd_cgo.go
 D vendor/github.com/tonistiigi/fsutil/diskwriter_openbsd.go

```